CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [v2.3.2](https://github.com/nextcloud-libraries/nextcloud-vite-config/compare/v2.3.1...v2.3.2) (2025-03-12)
+### Fixed
+* fix: Incomplete string escaping or encoding by @susnux in https://github.com/nextcloud-libraries/nextcloud-vite-config/pull/501
+* fix: Do not hide sourcemaps in production build by @juliusknorr in https://github.com/nextcloud-libraries/nextcloud-vite-config/pull/546
```

Comment:
```suggestion
* fix: Incomplete string escaping or encoding [\#501](https://github.com/nextcloud-libraries/nextcloud-vite-config/pull/501) \([susnux](https://github.com/susnux)\)
* fix: Do not hide sourcemaps in production build [\#546](https://github.com/nextcloud-libraries/nextcloud-vite-config/pull/546) \([juliusknorr](https://github.com/juliusknorr)\)
```

Reply:
Any tool used for that in the past? Will adjust manually for now but would be interesting if there is one that is regularly used that already comes up with this pattern